lib/retrieval-stats.js

```diff
@@ -39,10 +39,7 @@ export const buildRetrievalStats = (measurements, telemetryPoint) => {
   /** @type {Partial<Record<import('./typings.js').RetrievalResult, number>>} */
   const resultBreakdown = {
     OK: 0,
-    TIMEOUT: 0,
-    CAR_TOO_LARGE: 0,
-    BAD_GATEWAY: 0,
-    GATEWAY_TIMEOUT: 0
```

Comment:
why does this PR remove these 3 result codes from this test?

Reply:
Shouldn't there be new `LASSIE_` or `HTTP_` codes?

Reply:
See the comment above this object initialisation (GitHub diff does not show it by default, unfortunately):

```js
  // We are intentionally not initializing all possible keys here.
  // Example of omitted keys: UNDEFINED, ERROR_500 and ERROR_404.
  // The idea is that if we don't explicitly initialise them here and there is no measurement with
  // such retrieval result, then the Grafana dashboard will not show these results at all.
```

IIRC, I added a subset of keys to ensure they always show at the top of the legend in Grafana, and that we show these keys (retrieval result codes) even if no measurement reported the key.

<img width="373" alt="Screenshot 2024-10-23 at 13 31 05" src="https://github.com/user-attachments/assets/3dee83c2-d1c4-43bd-9c70-c76635171420">

I feel this is no longer needed/useful for all of these codes.

On the second thought, maybe `TIMEOUT` is worth keeping.

WDYT?

Reply:
>ensure they always show at the top of the legend in Grafana

This is no longer relevant. We have a transformation in Grafana to (re)order the codes manually.

Reply:
got it!